openapi/bootstrap-agent.json

```diff
@@ -77,6 +110,13 @@
           "request_id"
         ]
       },
+      "Ipv6Net": {
+        "title": "An IPv6 subnet",
```

Comment:
Sort of a nit, but this is not an arbitrary IPv6 subnet, it has to follow the scheme in RFD 63 sec 6.3. Might be worth a new type here, since it's only within the sled agent crate.

Reply:
I went ahead and added a wrapper type here, but it still wraps the underlying Ipv6Net type. As a heads-up, this also required manually implementing `Deserialize`, as I needed a way to validate the "subnet properties" distinguishing this from an arbitrary subnet.

Reply:
Ugh -- I forgot about needing to manually implement `Deserialize` and / or `JsonSchema`, sorry about that. Maybe we should use a `[u8; 8]` or something else that's literally just the prefix bytes. It's probably not worth it, but it's just annoying to have to write these traits by hand. Anyway, thanks for the update.